SpatialGDK/Source/SpatialGDK/Private/Utils/EntityFactory.cpp

```diff
@@ -385,6 +397,16 @@ TArray<FWorkerComponentData> EntityFactory::CreateEntityComponents(USpatialActor
 	return ComponentDatas;
 }
 
+void EntityFactory::CreatePopulateSkeletonComponents(USpatialActorChannel& Channel, TArray<FWorkerComponentData>& OutComponentCreates,
```

Comment:
The naming here is all bit hard to read. Populate and Create are both used as nouns in this function.
Is this meant to CreateActorComponents or PopulateSkeletonEntity?

OutComponentCreates probably doesn't indicate more than just OutComponents

Reply:
The important bit is the `OutComponentUpdates`, which we need to distinguish from `OutComponentCreates`. 

The naming is up for discussion, would `CreateSkeletonEntityPopulationComponents` show the method's purpose better?